tests/test_overfocus.py

```diff
@@ -52,73 +52,179 @@ def test_get_transformation_matrix(params):
     # params are relative to default parameters in function below
     'params', [
         (
-            {},
+            {
+                'overfocus': 1,
+                'scan_pixel_size': 1,
+                'camera_length': 1,
+                'detector_pixel_size': 1,
+                'cy': 0,
+                'cx': 0,
+                'y_px': 0.,
+                'x_px': 0.,
+                'fov_size_y': 0,
+                'fov_size_x': 0,
```

Comment:
What does it mean to have `fov_size_{x,y}=0`? Shouldn't the scan have at least one scan pixel?